ner4opt/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+:Version: 1.0.0 of April 9, 2023
+This module defines the public interface of the
+**Ner4Opt Library** providing access to the following modules:
+    - ``Ner4Opt``
+
+Example Usage
+~~~~~~~~~~~~~
+
+>>> from ner4opt import Ner4Opt
+>>> example_problem = "How much should the firm allocate in each asset so as to maximize its average return ?"
+>>> ner4opt = Ner4Opt('hybrid')  # select a model
```

Comment:
change Ner4Opt(model="hybrid") and the remove # comment

Reply:
In the comment part, you can instead add the possible string options ["lexical", .. "]